core/modules/user/user.admin.inc

```diff
@@ -354,7 +358,7 @@ function user_login_settings($form, &$form_state) {
   );
   $form['user_limit_settings']['help'] = array(
     '#type' => 'item',
-    '#description' => t('Configure the limit and the time window for users attempting to log in. That is, how many failed login attempts are allowed per specified time period.'),
+    '#description' => t('Do not allow any further login attempts <em>for the same user account</em>, if the limit has been reached. This is used to prevent repeated malicious login attempts (for example someone trying to guess the password of a specific user account).'),
```

Comment:
```suggestion
    '#description' => t('Do not allow any further login attempts <em>for the same user account</em> if the limit has been reached. This setting is used to prevent repeated malicious login attempts, for example, someone trying to guess the password of a specific user account.'),
```